package.json

```diff
@@ -23,7 +23,9 @@
     "d3-time": "1.0.x",
     "d3-time-format": "2.1.x",
     "eslint": "5.2.0",
+    "eslint-plugin-lodash": "^6.0.0",
```

Comment:
I'd like to remove this, I don't think we're going to get much value from it, and I'm keen to keep our dependency base as small as possible.

Reply:
done